src/App.js

```diff
@@ -16,6 +17,7 @@ export default () => (
         <Box p="24px">
           <Route path="/:entity" component={Entity} />
```

Comment:
We probably want to use `react-router` and only show the SpeakerForm when the visitor has hit the route `/speaker/submit` (Or a similar url if you can think of a better one!), could you update the PR to do that?

Reply:
Done, I think! Let me know if more changes needed.